modules/ROOT/pages/auradb/managing-databases/database-actions.adoc

```diff
@@ -15,24 +15,11 @@ To rename an instance:
 . Enter a new name for the instance.
 . Select *Rename*.
 
-== Loading a dataset
```

Comment:
You can no longer do this?

Reply:
nope, datasets were deleted from console, now a workspace thing.